Migrate contract package of iexec-common library (#3)
iExecBlockchainComputing/iexec-commons-poco: CHANGELOG.md

### New Features
- Init Gradle project. (#1 #2)

> - Migrate contract package of `iexec-common` library and the other classes used in the POCO protocol to this `iexec-commons-poco` library. (#3) 

Reviewer: Very minor
```suggestion
- Migrate from `iexec-common` library to this `iexec-commons-poco` library packages related to blockchain and iExec PoCo protocol. (#3) 
```
You can keep yours or another one no problem :+1:

Author: ok,it's integrated
